product_docs/docs/lasso/4/describe.mdx

```diff
@@ -4,2237 +4,2092 @@ navTitle: Gathered data details
+**Report output:**
 
-**Depth:** surface
+  * File `/tools/efm/systemd/service_name_cat.data`: Output of `systemctl cat service_name`
+  * File `/tools/efm/systemd/service_name_status.data`: Output of `systemctl status service_name`
 
+**Depth:** Surface
 
-**Security Impact:** *low* &mdash; 
-no known security impact.
+**Security impact:** Low &mdash; 
+No known security impact.
 
 ### etcd CLI (`etcd_cli`)
 
-Gathers the output of some `etcdctl` commands, if `etcdctl` is
-available in the server. The commands are: endpoint status and
-endpoint health.
+Gathers the output of some etcdctl commands, if etcdctl is
+available in the server. The commands are `endpoint status` and
+`endpoint health`.
 
-Report output:
+**Report output:**
 
-  * file `/tools/etcd/cli/endpoint_status.out`: Output of `etcdctl endpoint status` command
-  * file `/tools/etcd/cli/endpoint_health.out`: Output of `etcdctl endpoint health` command
+  * File `/tools/etcd/cli/endpoint_status.out`: Output of `etcdctl endpoint status` command
+  * File `/tools/etcd/cli/endpoint_health.out`: Output of `etcdctl endpoint health` command
 
-**Depth:** surface
+**Depth:** Surface
 
-
-**Security Impact:** *low* &mdash; 
-no known security impact.
+**Security impact:** Low &mdash; 
+No known security impact.
 
 ### etcd configuration (`etcd_configuration`)
 
-Collects etcd configuration file that is found in the server
-
-Report output:
+Collects `etcd` configuration file that's found in the server.
 
-  * file `/tools/etcd/config/basename`: etcd configuration file
+**Report output:**
 
-**Depth:** surface
+  * File `/tools/etcd/config/basename`: `etcd` configuration file
 
+**Depth:** Surface
 
-**Security Impact:** *low* &mdash; 
-no known security impact.
+**Security impact:** Low &mdash; 
+No known security impact.
 
 ### etcd systemctl (`etcd_systemctl`)
 
 When etcd services are detected, collects status and cat of the
-corresponding services. Will check for any service which name starts
-with `etcd`
-
-Report output:
+corresponding services. Checks for any service whose name starts
+with `etcd`.
 
-  * file `/tools/etcd/systemd/service_name_cat.data`: Output of 'systemctl cat service_name
-  * file `/tools/etcd/systemd/service_name_status.data`: Output of 'systemctl status service_name
+**Report output:**
 
-**Depth:** surface
+  * File `/tools/etcd/systemd/service_name_cat.data`: Output of 'systemctl cat service_name
+  * File `/tools/etcd/systemd/service_name_status.data`: Output of 'systemctl status service_name
 
+**Depth:** Surface
 
-**Security Impact:** *low* &mdash; 
-no known security impact.
+**Security impact:** low &mdash; 
+No known security impact.
 
 ### HARP CLI (`harp_cli`)
 
-Gathers output of a few `harpctl` command outputs using the
-`config.yml` file which is found in the server. The commands are:
-cluster, proxies, locations, nodes, and version.
+Gathers output of a few harpctl command outputs using the
```

Comment:
Inconsistent formatting: `harpctl` is a CLI tool with various commands, which would be invoked via the syntax `harpctl <command>`, e.g. `harpctl version` - most other such commands listed here are formatted as code, so this command should be as well.

Note: the command itself describes this inaccurately, and this inaccuracy is echoed below in the report output; the command issued *will be* of the form `harpctl ...`, not `harp ...`